Training .js

```diff
@@ -0,0 +1,35 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const trainingSchema = new Schema({
```

Comment:
Add a description on what this schema is about. Should the naming be TrainingSchema?

Reply:
Filename should be TrainingSchema too?